.github/workflows/pr-custom-review.yml

```diff
@@ -0,0 +1,27 @@
+name: Check reviews
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+
+jobs:
+  pr-custom-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-custom-review
+        uses: paritytech/pr-custom-review@v2
+        with:
+          token: ${{ secrets.PRCR_TOKEN }}
+          # 🔒 PROTECTED: Changes to locks-review-team should be approved by the current locks-review-team
+          locks-review-team: locks-review
+          team-leads-team: polkadotleads
+          action-review-team: ci
```

Comment:
Every input except `token` should be removed for `v2` (those inputs are in `.github/pr-custom-review.yml` now)

```suggestion
```